problems.md

```diff
@@ -14,75 +14,67 @@ The actual source of truth of problem order is [config.json](config.json), but t
 
 The first section contains the sort of stuff you expect when learning any programming languages: conditionals, booleans, looping and some higher-order functions.
 
-problem | topics
------ | -----
-hello-world |  Some/None. println!
-gigasecond |  crates, math
-leap |  math, booleans, conditionals
-raindrops |  case (or `format`). Mutable string
-bob |  chars, string functions
-beer-song |  case, string concatenation, vector (optional), loop
-difference-of-squares |  fold & map
-sum-of-multiples | algorithm, borrowing
-grains | math, panic
+* hello-world
```

Comment:
Or alternatively, would "This section starts at `hello-world` and ends at `grains`" be acceptable? Might be less maintenance/duplication that way?